nomad/acl_endpoint.go

```diff
@@ -640,6 +638,14 @@ func (a *ACL) GetToken(args *structs.ACLTokenSpecificRequest, reply *structs.Sin
 				return err
 			}
 
+			// Check management level permissions or that the secret ID matches the
+			// accessor ID
+			if acl != nil && out != nil {
```

Comment:
Do we need to return PermissionDenied if `out == nil && !mgmt` to prevent leaking whether or not the AccessorID exists?